templates/firewall/resources/vkcs_networking_secgroup_rule.md.tmpl

```diff
@@ -10,9 +10,15 @@ description: |-
 
 {{ .Description }}
 
-## Example Usage
+## Example Usage one protocol one port
```

Comment:
"Example Usage" is a chapter name. Add example names on the next level. See compute examples

Reply:
Name it "Basic rule"